builder/parallels/common/driver_9.go

```diff
@@ -232,6 +232,18 @@ func (d *Parallels9Driver) Stop(name string) error {
 	return nil
 }
 
+// Reboot softly reboots the VM.
+func (d *Parallels9Driver) Reboot(name string) error {
+	if err := d.Prlctl("restart", name); err != nil {
+		return err
+	}
+
+	// We sleep here for a little bit to let the session "unlock"
+	time.Sleep(2 * time.Second)
```

Comment:
Is there any way we can check an actual lock file instead of sleeping for an arbitrary length of time? If this is an issue in Parallels, is it an issue in other options?

Reply:
I am not familiar with Parallels, I was simply writing the minimal code to propagate this new `boot_command` directive to the various VM implementations.